draft-ietf-quic-http.md

```diff
@@ -1076,7 +1076,9 @@ QPACK. See [QPACK] for more details.
 ~~~~~~~~~~
 {: #fig-headers title="HEADERS frame payload"}
 
-HEADERS frames can only be sent on request / push streams.
+HEADERS frames can only be sent on request / push streams.  If a HEADERS frame
+is received on either control stream, the recipient MUST respond with a
```

Comment:
Maybe "on _a_ control stream"? I think that we do not normally talk about two control streams (in each direction) differently.

Reply:
Wouldn't `on the control stream` be better then? As far as I'm aware we still have only one of those.